src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+    private String color = getColor();
```

Comment:
There's no need to initialize the 'color' variable here since it's being set in the constructor. You can remove this initialization.